web3rpc/rpc-specs/paths/klay/miscellaneous/getActiveAddressFromRegistry.yaml

```
@@ -0,0 +1,100 @@
+        - klay
+
+      parameters:
+        - name: name
+          required: true
+          schema:
+            type: string
+          description: Name of which active system contract registered with.
+          in: query
+        - name: blockNumberOrHashOrTag
+          in: query
+          description: Integer or hexadecimal block number, or the string "earliest", "latest" or "pending" as in the default block parameter, or block hash.
+          required: true
+          schema:
+            $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: "../../../components/requests/JsonRpcRequest.yaml#/components/schemas/JsonRpcRequest"
+                - $ref: "#/components/schemas/KlaygetActiveAddressFromRegistryReq"
+
+      responses:
+        200:
+          description: Returns the account key or an empty key value.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "../../../components/responses/JsonRpcResponse.yaml#/components/schemas/JsonRpcResponse"
+                  - $ref: "#/components/schemas/KlaygetActiveAddressFromRegistryResp"
+
+      x-codeSamples:
+        - lang: "curl"
+          label: "Curl"
+          source:
+            $ref: "../../../code-samples/curl/klay/miscellaneous/getActiveAddressFromRegistry.sh"
+        - lang: "java"
+          label: "Java"
+          source:
+            $ref: "../../../code-samples/java/src/main/java/opensdk/sdk/apis/klay/miscellaneous/KlayGetActiveAddressFromRegistryExample.java"
+        - lang: "javascript"
+          label: "Javascript"
+          source:
+            $ref: "../../../code-samples/javascript/klay/miscellaneous/getActiveAddressFromRegistry.js"
+        - lang: "python"
+          label: "Python"
+          source:
+            $ref: "../../../code-samples/python/klay/miscellaneous/getActiveAddressFromRegistry.py"
+
+components:
+  schemas:
+    KlaygetActiveAddressFromRegistryReq:
+      type: object
+      required:
+        - method
+        - params
+      properties:
+        method:
+          type: string
+          default: 'klay_getActiveAddressFromRegistry'
+        params:
+          type: array
+          items:
+            properties:
+              KipType:
+                  type: string
+                  example: kip113
+              BlockNumberOrHashOrTag:
+                $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+          example:  ["kip113", "latest"]
+
+    KlaygetActiveAddressFromRegistryResp:
+      type: object
+      properties:
+        result:
+            type: string
+            description: Returns address of active system contract registered with name if exists.
+            example: "0x0000000000000000000000000000000000000402"
```

The arguments are case-sensitive. It must be `KIP113` to work.

```suggestion
          example:  ["KIP113", "latest"]

    KlaygetActiveAddressFromRegistryResp:
      type: object
      properties:
        result:
            type: string
            description: Returns address of active system contract registered with name if exists.
            example: "0x3e80e75975bdb8e04b800485dd28bebec6d97679"
```